src/osdc/Objecter.h

```diff
@@ -1658,6 +1658,8 @@ class Objecter : public md_config_obs_t, public Dispatcher {
   std::atomic<int> global_op_flags{0}; // flags which are applied to each IO op
   bool keep_balanced_budget = false;
   bool honor_pool_full = true;
+  std::atomic<int> num_op_laggy{0};
+  std::atomic<int> num_osd_laggy{0};
```

Comment:
For this issue, it seems there is no need to track both the laggy ops and OSDs at the same time, and just the laggy OSDs should be enough.

From the `Objecter.cc` code, once there has any laggy ops, there must have at least one laggy OSD.

Reply:
yeah makes sense, will update it. thanks!

Reply:
could there be any edge case where we have laggy ops but not laggy osds?

Reply:
From the code, I didn't see that.

Reply:
sounds good then. thanks again

Reply:
removed function to fetch laggy ops